content/metadata-rework.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "Metadata rework"
+date = 2024-07-25
+weight = 39
+template = "page.html"
+render = true
++++
+
+*John Baublitz, Stratis Team*
+
+Until now, Stratis has had a metadata format that, with a few exceptions, works
+remarkly well for our current feature set. However, as we plan for larger features
+like RAID, integrity, and online reencryption, we've noticed that we require some
+reworks to our stacking and the addition of reservation ahead of time for new devicemapper
+layers. The metadata rework has been a large effort to pave the way for Stratis moving forward
+and will provide the ability for additional flexibility in enabling and disabling certain
+layers after pool creation time once the features are added.
+
+### Changes to encryption
+
+Encryption might be the single largest change in the metadata rework PR. The crypt device now
```

Comment:
^Encryption ... .^The changes in the way Stratis support encryption may be the most significant.

Otherwise, to people who haven't been or aren't paying attention, it looks like this change add encryption.

Reply:
Resolved.